### PR TITLE
feat: Added the alert when no results found

### LINKED ISSUE #89: [FEATURE]: Add a message when No results found after applying the filter
### Description

A message box needs to be created when no results found after applying filters

### Screenshots

Screenshot (298)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by RajeevDash007**
## Fixes Issue
closes #89 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (299)

## Note to reviewers
Please let me know , if you need some more changes

### CODE REVIEW

**Comment by RajeevDash007** [comment]
@eddiejaoude have a look at this.

File: src/app/search/page.js [hunk 1]
```diff
@@ -57,35 +58,46 @@ export default async function Page({ searchParams }) {
...
+                      tags: user.tags?.split(","),
+                    }))}
+                  />
+                ) : (
+                  <Alert
+                    type="info"
+                    message="No results found"
+                    details={[
+                      "Try adjusting your search parameters.",
+                      "Check for typos or broaden your search criteria.",
```
Comment (eddiejaoude) [inline src/app/search/page.js:97]: There are no ways for the user to have types as they are all drop downs

```suggestion
                      "Try to broaden your search criteria.",
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you 👍 Looks good, I left 1 inline comment with a suggestion

**Comment by RajeevDash007** [comment]
Cool!!! 
I will resolve it and let you know.
Thanks

**Comment by RajeevDash007** [comment]
@eddiejaoude done👍
Lemme know if you need some more changes

File: src/app/search/page.js (hunk 1 above)
Comment (eddiejaoude) [reply src/app/search/page.js]: Changes look better. BTW for the future you can press accept on the suggestion ☝️ instead of writing it again

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

File: src/app/search/page.js (hunk 1 above)
Comment (RajeevDash007) [reply src/app/search/page.js]: Okay, thank you so much  for letting me know.
I will keep it in my mind .

**Pull request merged**